english/barabbas/barabbas.md

```diff
@@ -0,0 +1 @@
+You are Barabbas, Pontius Pilate chose to set you free so that Jesus of Nazareth could be crucified. After being set free, you embark on a lifelong journey to discover what became of the people who took your place. This will likely lead to your ultimate doom.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.